haproxy/README.md

```diff
@@ -94,4 +118,15 @@ Set `haproxy.config` to `galerak8s` to use the Galera configuration with DNS ser
 | `haproxy.galerak8s.nodeCount` | Max number of nodes in the backend | 3
 | `haproxy.galerak8s.port` | Port of the Galera node | 3306
 
+### redisk8s
+
+Set `haproxy.config` to `redisk8s` to use the Redus configuration with DNS service discovery. See https://www.haproxy.com/de/blog/dns-service-discovery-haproxy/
+
+| Parameter              | Description            | Default
+|---                     | ---                    | ---
+| `haproxy.redisk8s.check.enabled` | If check should be enabled | true
+| `haproxy.redisk8s.check.redis.auth` | If Authentication credentials for Redis | 
```

Comment:
Would be nice to be able to reference an external secret here as well.

Reply:
see haproxy.check.existingSecret